app/services/account_search_service.rb

```diff
@@ -151,13 +151,23 @@ def core_query
   end
 
   def call(query, account = nil, options = {})
-    @query   = query&.strip&.gsub(/\A@/, '')
-    @limit   = options[:limit].to_i
-    @offset  = options[:offset].to_i
-    @options = options
-    @account = account
+    MastodonOTELTracer.in_span('AccountSearchService#call') do |span|
```

Comment:
Maybe premature with just a few uses here, but if this pattern grows more, we could extract something like an `tracer do |span|...` sort of method here could reflect on where it's being called from to create the span label. Could be just in service base class for now, and move elsewhere if used beyond services.

Reply:
That would make the code-in-place cleaner to read and there's definitely some noisiness to instrumentation.

Some things to consider before abstraction:

1. There's some troubleshooting power in seeing a span in a trace named `AccountSearchService#call` and being able to grep the codebase for `/span\(["']AccountSearchService#call["']/` to quickly find the location of the span creation.
2. Reflection requires compute. Minuscule, but it can add up. In some of the hot-paths in the OTel libraries, profiling has conditioned us to avoid even method dispatch if possible.